src/node_options.cc

```diff
@@ -1540,7 +1588,8 @@ void Initialize(Local<Object> target,
       context, target, "getCLIOptionsInfo", GetCLIOptionsInfo);
   SetMethodNoSideEffect(
       context, target, "getEmbedderOptions", GetEmbedderOptions);
-
+  SetMethodNoSideEffect(
```

Comment:
This function has side effects

```suggestion
  SetMethod(
```